### ISSUE TITLE
Missing `@pnpm/logger` dependency (peer dependency of `@pnpm/lockfile-file`)

### ISSUE THREAD

**Issue by macksal** [0x80/isolate-package#31]
I'm running `isolate` in a yarn (v1) workspace monorepo, and getting `Error: Cannot find module '@pnpm/logger'`. It seems this is a peer dependency of `@pnpm/lockfile-file`, which is a dependency of `isolate-package`.

It's not listed as a dependency in this repo, so it never ends up being installed in mine.

It seems like a simple fix to add `@pnpm/logger@^5.0.0` as a dependency in this repo.

**Comment by macksal** [comment]
Maybe having pnpm installed exposes the @pnpm apis globally? (Or it's leftover in your environment from something else)

**Comment by 0x80** [comment]
Yes, something like that 🤷  I have released 1.5.1 just now, that should fix it!
